packages/design-system/src/ListSegmentHeader/index.tsx

```diff
@@ -34,9 +26,9 @@ export default function SegmentHeader(props: SegmentHeaderProps) {
       style={props.style}
     >
       {props.title}
-      {!props.hideStyledHr && (
-        <StyledHr data-testid={"t--styled-segment-header-hr"} />
```

Comment:
Why this is removed?

Reply:
There are no places where this is being used in the main repo right now. I was getting an error on local with this prop and removing this fixed it.